entity-framework/core/modeling/bulk-configuration.md

```diff
@@ -0,0 +1,65 @@
+- Entity types might be added after this iteration happens and the configuration won't be applied to them. This can usually be prevented by placing this code at the end of `OnModelCreating`, but if you have two interdependent sets of configurations there might not be an order that will allow them to be applied consistently.
+
+## Pre-convention configuration
+
+EF Core 6.0 allows the mapping configuration to be specified once for a given type. It will then be applied to all properties of that type in the model as they are discovered. This is called "pre-convention model configuration", since it configures aspects of the model that are then used by the model building conventions. Such configuration is applied by overriding `<xref:Microsoft.EntityFrameworkCore.DbContext.ConfigureConventions>` on the type derived from `<xref:Microsoft.EntityFrameworkCore.DbContext>`:
+
+[!code-csharp[Main](../../../samples/core/Modeling/BulkConfiguration/PreConventionContext.cs?name=MoneyConversion)]
+
+[!code-csharp[Main](../../../samples/core/Modeling/BulkConfiguration/PreConventionContext.cs?name=StringFacets)]
+
+> [!NOTE]
+> The type specified in a call from `ConfigureConventions` can be a base type, an interface or a generic type definition. All matching configurations will be applied in order from the least specific:
+>
+> 1. Interface
+> 2. Base type
+> 3. Generic type definition
+> 4. Non-nullable value type
+> 5. Exact type
+
+### Ignoring types
+
+Pre-convention configuration also allows to ignore a type and prevent it from being discovered by conventions either as an entity type or a entity type property:
+
+[!code-csharp[Main](../../../samples/core/Modeling/BulkConfiguration/PreConventionContext.cs?name=IgnoreInterface)]
+
+### Default type mapping
+
+Generally, EF will be able to translate queries with constants of a type that is not supported by the provider if you have specified a value converter for a property of this type. However, in the queries that don't involve any properties of this type there is no way for EF to find the correct value converter. In this case it's possible to call <xref:Microsoft.EntityFrameworkCore.ModelConfigurationBuilder.DefaultTypeMapping> to add or override a provider type mapping:
+
+[!code-csharp[Main](../../../samples/core/Modeling/BulkConfiguration/PreConventionContext.cs?name=DefaultTypeMapping)]
+
+### Limitations of pre-convention configuration
+
+- Many aspects cannot be configured with this approach. [#6787](https://github.com/dotnet/efcore/issues/6787) will expand this to more types.
+- Currently the configuration is only determined by the CLR type. [#20418](https://github.com/dotnet/efcore/issues/20418) would allow custom predicates.
+- This configuration is performed before a model is created, if there are any conflicts that arise when applying it the exception stack trace will not contain the `ConfigureConventions` method, so it might be harder to find the cause.
```

Comment:
```suggestion
- This configuration is performed before a model is created. If there are any conflicts that arise when applying it, the exception stack trace will not contain the `ConfigureConventions` method, so it might be harder to find the cause.
```